src/System/TimedScopes/TimedScopeEventSource.cs

```diff
@@ -0,0 +1,233 @@
+﻿/***************************************************************************************************
+	TimedScopeEventSource.cs
+
+	Copyright (c) Microsoft Corporation.
+
+	Timed scopes event source
+***************************************************************************************************/
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Omex.System.Logging;
+using Microsoft.Omex.System.TimedScopes;
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// Timed scopes event source
+	/// </summary>
+	[EventSource(Name = "Microsoft-OMEX-TimedScopes")]
+	public sealed class TimedScopeEventSource : EventSource
+	{
+		#region Initialization
+
+		/// <summary>
+		/// An initialized instance of the <see cref="TimedScopeEventSource"/> class
+		/// The singleton behavior is needed so that the Event Source works properly
+		/// </summary>
+		public static TimedScopeEventSource Instance => s_instance.Value;
+
+
+		/// <summary>
+		/// Static constructor
+		/// </summary>
+		static TimedScopeEventSource()
+		{
+			// A workaround for the problem where ETW activities do not get tracked until Tasks infrastructure is initialized.
+			// This problem will be fixed in .NET Framework 4.6.2.
```

Comment:
Do we support these old versions of .NET?

Reply:
The README.md says we only support .NET Core 3.0+ now

Reply:
If I'm understanding correctly README.md talking about SDK version required to build the project, but `Microsoft.Omex.System` string targets lower frameworks and used in projects that target net46

Reply:
Thanks for clarifying.